plugins/sigma/tests/tests.cpp

```diff
@@ -158,7 +158,25 @@ TEST(modifier - re) {
     foo|re: "^.*$"
   )__";
   auto search_id = to_search_id(yaml);
-  auto expected = to_expr("foo == /^.*$/");
+  auto expected = to_expr("foo == /^.*$/i");
+  CHECK_EQUAL(search_id, expected);
+}
+
+TEST(modifier - re 2) {
+  auto yaml = R"__(
+    foo|re: '.*foobar.*'
```

Comment:
I would argue that this should instead map to `/foobar/ in foo` (minus operand flipping).

Why? Because the regex can be optimized more easily.